enterprise/backend/src/metabase_enterprise/serialization/v2/models.clj

```diff
@@ -51,7 +51,7 @@
    "HTTPAction"
    "ImplicitAction"
    "LoginHistory"
-   "MetricImportantField"
```

Comment:
We might actually be able to just drop MetricImportantField, I'm not sure this is even used anymore TBH. If it is we can probably sunset that immediately since Metrics v1 are going away anyway